src/btree/bt_vrfy.c

```diff
@@ -370,7 +370,7 @@ __wt_verify(WT_SESSION_IMPL *session, const char *cfg[])
 err:
     /* Inform the underlying block manager we're done. */
     if (bm_start)
-        WT_TRET(bm->verify_end(bm, session));
+        WT_TRET(bm->verify_end(bm, session, ret));
```

Comment:
If we go ahead with the boolean change, it would make sense to do
```suggestion
        WT_TRET(bm->verify_end(bm, session, ret == 0));
```
Or
```suggestion
        WT_TRET(bm->verify_end(bm, session, ret != 0));
```
Depending on the logic we want to go for.

Reply:
boolean makes sense to me :)

Reply:
Changed to `verify_success` in [453d2c7](https://github.com/wiredtiger/wiredtiger/pull/12447/commits/453d2c7e22c9981b8b42c899542e99738096e3e9)